homeassistant/components/sensor/opensky.py

```diff
@@ -0,0 +1,141 @@
+"""
+Sensor for the Open Sky Network.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.opensky/
+"""
+import logging
+from datetime import timedelta
+
+import requests
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_NAME, CONF_LATITUDE, CONF_LONGITUDE,
+    ATTR_ATTRIBUTION, ATTR_LATITUDE, ATTR_LONGITUDE,
+    LENGTH_KILOMETERS, LENGTH_METERS)
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import distance as util_distance
+from homeassistant.util import location as util_location
+import homeassistant.helpers.config_validation as cv
+
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=12)  # opensky public limit is 10 seconds
```

Comment:
I think that we should be friendly and not being too close to the limit of the API. Isn't it enough to get the data every 30 or so?